sql/core/src/test/resources/sql-tests/inputs/pipe-operators.sql

```diff
@@ -665,15 +665,28 @@ table t
 table t
 |> order by x sort by x;
 
--- The WINDOW clause is not supported yet.
+-- The WINDOW clause: positive tests.
+-------------------------------------
 table windowTestData
-|> window w as (partition by cte order by val)
+|> window w as (partition by cate order by val)
 |> select cate, sum(val) over w;
 
+table windowTestData
+|> window w1 as (partition by cate), w2 as (partition by val order by cate)
+|> select sum(val) over w1, first_value(cate) over w2;
```

Comment:
Other test ideas:

* `|> window ...` with RANGE BETWEEN as part of the window definition
* what happens if we add a `|> window` operator, but then we do not refer to it in the following `|> select` operator (using `over`)?
* `|> window ...` with other operators after it, e.g. `|> where` or `|> order by`
* `|> window ...` creating window names that shadow other valid column names. Is this allowed?
* `|> window ...` using table aliases to refer to columns, like L193-207 above
* `|> window ...` with nothing after it at all
* `|> window ...` and OFFSET at the same time
* `|> window ...` and ORDER BY at the same time, but outside the parentheses (e.g. `|> window w1 as (partition by val) order by cate`)
* `|> select ...` with inline windows not defined in a preceding `|> window ...` operator

Reply:
> |> window ... creating window names that shadow other valid column names. Is this allowed?

It is allowed in the current SQL syntax (see the pic). 

Do you know if this is expected? If not, we may need a separate issue to fix this.

![image](https://github.com/user-attachments/assets/70758c2d-c64a-4afa-827d-c97ef7cd6c76)

Reply:
If it's allowed in regular SQL, we should probably allow it the same way here for consistency. It looks like we are exercising that in tests now and it works.